{
    "namespaces": {
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": [
                    {
                        "children": [],
                        "cpe_match": [
                            {
                                "cpe23Uri": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                "cpe_name": [],
                                "versionEndExcluding": "3.2.16",
                                "versionStartIncluding": "3.2",
                                "vulnerable": true
                            },
                            {
                                "cpe23Uri": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                "cpe_name": [],
                                "versionEndExcluding": "4.1.2",
                                "versionStartIncluding": "4.1",
                                "vulnerable": true
                            },
                            {
                                "cpe23Uri": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                "cpe_name": [],
                                "versionEndExcluding": "4.0.8",
                                "versionStartIncluding": "4.0",
                                "vulnerable": true
                            }
                        ],
                        "operator": "OR"
                    }
                ]
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2022-41323"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "In Django 3.2 before 3.2.16, 4.0 before 4.0.8, and 4.1 before 4.1.2, internationalized URLs were subject to a potential denial of service attack via the locale parameter, which is treated as a regular expression."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "en",
                                    "value": "NVD-CWE-Other"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://docs.djangoproject.com/en/4.0/releases/security/",
                            "refsource": "MISC",
                            "tags": [
                                "Release Notes",
                                "Vendor Advisory"
                            ],
                            "url": "https://docs.djangoproject.com/en/4.0/releases/security/"
                        },
                        {
                            "name": "https://github.com/django/django/commit/5b6b257fa7ec37ff27965358800c67e2dd11c924",
                            "refsource": "MISC",
                            "tags": [
                                "Patch",
                                "Third Party Advisory"
                            ],
                            "url": "https://github.com/django/django/commit/5b6b257fa7ec37ff27965358800c67e2dd11c924"
                        },
                        {
                            "name": "https://www.djangoproject.com/weblog/2022/oct/04/security-releases/",
                            "refsource": "CONFIRM",
                            "tags": [
                                "Vendor Advisory"
                            ],
                            "url": "https://www.djangoproject.com/weblog/2022/oct/04/security-releases/"
                        },
                        {
                            "name": "https://groups.google.com/forum/#!forum/django-announce",
                            "refsource": "MISC",
                            "tags": [
                                "Issue Tracking",
                                "Third Party Advisory"
                            ],
                            "url": "https://groups.google.com/forum/#!forum/django-announce"
                        }
                    ]
                }
            },
            "impact": {
                "baseMetricV3": {
                    "cvssV3": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 7.5,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "NONE",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
                        "version": "3.1"
                    },
                    "exploitabilityScore": 3.9,
                    "impactScore": 3.6
                }
            },
            "lastModifiedDate": "2022-10-18T19:54Z",
            "publishedDate": "2022-10-16T06:15Z"
        }
    }
}